tests/test_install.py

```diff
@@ -183,24 +183,20 @@ def tag(self, name, message):
 
 
 def check_version(version, exp_version, dirty=False):
-    dirty_tag = ".d" + strftime("%Y%m%d")
+    today = datetime.now(timezone.utc).strftime("%Y%m%d")
```

Comment:
[Python docs](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) recommend using explicit timezone instead of `utcnow`.

Reply:
Thanks, didn't know that.